tests/test_cross_compile.py

```diff
@@ -221,11 +226,14 @@ def test_cross_python_no_build(tmp_path):
 
 
 @flaky
-def test_build2host(tmp_path):
+@pytest.mark.parametrize("recipe_version", [0, 1])
+def test_build2host(recipe_version, tmp_path):
```

Comment:
I think we might want flaky inside the pytest parameterize as opposed to outside of it.